src/compiler/file/ImportSpecifier.ts

```diff
@@ -29,6 +29,13 @@ export class ImportSpecifier extends Node<ts.ImportSpecifier> {
      * @param name - New name.
      */
     renameName(name: string) {
+        return this.rename(name);
+    }
+    /**
+     * Renames the identifier being imported.
+     * @param name - New name.
+     */
+    rename(name: string) {
```

Comment:
It's called `renameName` so there's no confusion about what's being renamed. Some people might expect a `rename` method to rename the alias.

Perhaps we should deprecate `renameName` and instead have this `rename` method rename the alias if it exists and rename the name if the alias doesn't exist. Would that make sense?